tests/services/instruments/test_dvt_secure_socket_proxy.py

```diff
@@ -49,6 +49,17 @@ def test_applist(dvt):
     assert safari['Type'] == 'PluginKit'
 
 
+def test_memlimitoff(dvt):
+    """
+    Test disabling memory limit.
+    """
+    aggregated = get_process_data(dvt, 'SpringBoard')
```

Comment:
Probably meant `springboard` for the variable? or did u actually plan to check with `SpringBoard`?